scripts/sn_flux_est.py

```diff
@@ -17,10 +17,10 @@ def snfe_main(kwargs):
         if kwargs["pointing"]:
```

Comment:
Actually on Line 7, can @NickSwainston or @mengyaoxue please confirm this is the correct module import statement. I had to make a decision to resolve a conflict. I'm pretty sure this is right, but you'd be the ones to know.

Reply:
It is wrong, should be:
```
from vcstools.pulsar_spectra import ATNF_spectral_data_plot
```

Reply:
Okay, updated.